overview.md

```diff
@@ -43,130 +41,180 @@ steps:
+# - bash: echo '##vso[task.prependpath]/usr/local/MATLAB/R2020b/bin'  # Linux agent
   - task: RunMATLABCommand@1
     inputs:
       command: myscript
 ```
 
-### Use MATLAB on Microsoft-Hosted Agent
-Before you run MATLAB code or Simulink models on a Microsoft-hosted agent, first use the [Install MATLAB](#install-matlab) task. The task installs your specified MATLAB release (R2021a or later) on a Linux&reg; virtual machine. If you do not specify a release, the task installs the latest release of MATLAB.
+### Install Transformation Product on Microsoft-Hosted Agent
+To use a transformation product, such as MATLAB Coder&trade; and MATLAB Compiler&trade;, on a Microsoft-hosted agent, you need a MATLAB batch licensing [token](https://github.com/mathworks-ref-arch/matlab-dockerfile/blob/main/alternates/non-interactive/MATLAB-BATCH.md#matlab-batch-licensing-token). Batch licensing tokens are strings that enable MATLAB to start in noninteractive environments. You can request a token by contacting MathWorks&reg; at [batch-tokens@mathworks.com](mailto:batch-tokens@mathworks.com).
+
+To use a MATLAB batch licensing token:
 
-For example, install MATLAB R2023b on a Microsoft-hosted agent, and then use the **Run MATLAB Command** task to run the commands in your script.
+1. Set the token as a secret variable. For more information about secret variables, see [Set secret variables](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/set-secret-variables?view=azure-devops&tabs=yaml%2Cbash).
+2. Map the secret variable into an environment variable named `MLM_LICENSE_TOKEN` in your YAML pipeline.
+
+For example, use the latest release of MATLAB Compiler on a Microsoft-hosted agent to create a standalone application using a file named `myfun.m` in the root of your repository. To install MATLAB and MATLAB Compiler on the agent, specify the **Install MATLAB** task in your pipeline. To run the function required for creating the application, specify the **Run MATLAB Command** task. In this example, `myToken` is the name of the secret variable that holds the batch licensing token.
 
 ```YAML
 pool:
   vmImage: ubuntu-latest
 steps:
-  - task: InstallMATLAB@1
-    inputs:
-      release: R2023b
-  - task: RunMATLABCommand@1
-    inputs:
-      command: myscript
+- task: InstallMATLAB@1
+  inputs:
+    products: MATLAB_Compiler
+- task: RunMATLABCommand@1
+  inputs:
+    command: compiler.build.standaloneApplication("myfun.m")
+  env:
+    MLM_LICENSE_TOKEN: $(myToken)
+```
+
+### Build Across Multiple Platforms
+The **Install MATLAB** task supports the Linux, Windows, and macOS platforms. Use a `matrix` job strategy to run a build using the MATLAB build tool on all the supported platforms. This pipeline runs three jobs.
+
+```YAML
+strategy:
+  matrix:
+    linux:
+      imageName: ubuntu-latest
+    mac:
+      imageName: windows-latest
+    windows:
+      imageName: macOS-latest
```

Comment:
It seems there might be a need to swap the image types between Mac and Windows.

Reply:
Good catch! Thanks, Tushar.